src/codechandler/AVCCodecHandler.h

```diff
@@ -0,0 +1,16 @@
+#include "CodecHandler.h"

```

Comment:
in this and many other .h headers is missing
`#pragma once` on top
there is a reason behind?

Reply:
Yes.
It's not required unless the file is going be be #included more than once.